wp-content/plugins/zippicks-smart-search/includes/class-api-client.php

```
@@ -0,0 +1,390 @@
+    public function check_connection() {
+        return $this->test_connection();
+    }
+    
+    /**
+     * Get autocomplete suggestions
+     * 
+     * @param string $prefix Search prefix
+     * @param array $location Location data
+     * @return array|WP_Error
+     */
+    public function get_autocomplete($prefix, $location) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$location'. (Unused Code Rules)

(UnusedFormalParameter)